src/providers/PortfolioProvider/hooks/useBalancesData.ts

```
@@ -0,0 +1,177 @@
+            balances: cached,
+            round: 0,
+            updatedAt: null,
+          };
+        }
+        return undefined;
+      },
+    })),
+  });
+
+  const refetch = useCallback(() => {
+    controlsRef.current.forEach((ctrl) => ctrl.cancel());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)